provisioning/provisioning.go

```diff
@@ -44,12 +44,12 @@ var (
 	findBootloader = partition.FindBootloader
```

Comment:
this whole package seems a 15.04 left over that we didn't spot yet, kill it here?